remotes/docker/resolver_test.go

```diff
@@ -68,11 +68,15 @@ func TestBasicResolver(t *testing.T) {
 		})
 
 		base, options, close := tlsServer(wrapped)
+		authorizer := NewDockerAuthorizer(
+			WithAuthClient(options.Client),
```

Comment:
Note that `options.Client` is also deprecated, in favour of .. erm. `options.Host` (which is what we're creating here) 😂)